taxi/views.py

```diff
@@ -85,5 +87,47 @@
 
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
-    model = Driver
+    model = get_user_model()
```

Comment:
You should use `get_user_model()` instead of `Driver`. Please, replace `Driver` with `get_user_model()`.